tests/integration/test_zosmf_workflows.py

```diff
@@ -0,0 +1,209 @@
+FILES_FIXTURES_PATH = os.path.join(FIXTURES_PATH, "files.json")
+SAMPLE_WORKFLOW_FIXTURE_PATH = os.path.join(FIXTURES_PATH, "workflow.xml")
+SAMPLE_WORKFLOW_INPUT_FIXTURE_PATH = os.path.join(FIXTURES_PATH, "workflow_input")
+
+
+class TestZosmfWorkflowsIntegration(unittest.TestCase):
+    """Workflows class integration tests."""
+
+    created_workflow: CreateWorkflowResponse = None
+
+    def setUp(self):
+        """Setup fixtures for Zosmf class."""
+        test_profile = ProfileManager(show_warnings=False).load(profile_type="zosmf")
+        self.workflows = Workflows(test_profile)
+        self.files = Files(test_profile)
+        self.addCleanup(lambda: self.workflows.__exit__(None, None, None))
+        self.addCleanup(lambda: self.files.__exit__(None, None, None))
+        with open(FILES_FIXTURES_PATH, "r") as fixtures_json:
+            self.files_fixtures = json.load(fixtures_json)
+        self.test_uss_workflow = self.files_fixtures["TEST_USS_WORKFLOW"]
+        self.test_uss_workflow_input = self.files_fixtures["TEST_USS_WORKFLOW_INPUT"]
+
+    def tearDown(self):
+        try:
+            self.files.uss.delete(self.test_uss_workflow)
+        except:
+            pass
+        try:
+            self.files.uss.delete(self.test_uss_workflow_input)
+        except:
+            pass
+
+        if self.created_workflow != None:
+            try:
+                self.workflows.delete_workflow(self.created_workflow.workflowKey)
+            except:
+                pass
+            try:
+                self.workflows.delete_archived_workflow(self.created_workflow.workflowKey)
+            except:
+                pass
+
+    def test_get_definition_create_get_properties_list_start_cancel_delete(self):
+        """Get workflow definition, create a workflow, get it's properties, list workflows by the name regex, start the workflow, cancel and delete it"""
+
+        self.files.uss.upload(SAMPLE_WORKFLOW_FIXTURE_PATH, self.test_uss_workflow, encoding="IBM-1047")
+        self.files.uss.upload(SAMPLE_WORKFLOW_INPUT_FIXTURE_PATH, self.test_uss_workflow_input, encoding="IBM-1047")
+
+        workflow_def = self.workflows.get_workflow_definition(
+            self.test_uss_workflow, return_steps_data=True, return_variables_data=True
+        )
+
+        self.assertIsInstance(workflow_def, GetWorkflowDefinitionResponse)
+        self.assertEqual(workflow_def.workflowDefaultName, "Zowe Test Workflow")
+        self.assertEqual(len(workflow_def.steps), 2)
+        self.assertEqual(len(workflow_def.variables), 3)
+
+        self.created_workflow = self.workflows.create_workflow(
+            workflow_name="Zowe Test Workflow (Common)",
+            workflow_definition_file=self.test_uss_workflow,
+            system="BT41",
+            owner="gast88n",
+            variable_input_file=self.test_uss_workflow_input,
+            workflow_archive_safid="gast88n"
```

Comment:
We should load these values from fixtures files.
I believe we might need something like the `.env.example` file where people can copy it and provide their own set of information.